libs/community/tests/unit_tests/vectorstores/test_aerospike.py

```diff
@@ -0,0 +1,378 @@
+import sys
+from typing import Any, Callable, Generator
+from unittest.mock import MagicMock, Mock, call
+
+import pytest
+from langchain_core.documents import Document
+
+from langchain_community.vectorstores.aerospike import Aerospike
+from langchain_community.vectorstores.utils import DistanceStrategy
+from tests.integration_tests.vectorstores.fake_embeddings import FakeEmbeddings
+
+pytestmark = pytest.mark.requires("aerospike_vector_search") and pytest.mark.skipif(
+    sys.version_info < (3, 9), reason="requires python3.9 or higher"
```

Comment:
fyi the aerospike-vector-search pkg is documented as supported py3.8 but given how things are typed in it (e.g. `dict[str]` style typing) it doesn't actually work with <py3.9. see example failure here: https://github.com/langchain-ai/langchain/actions/runs/9166787019/job/25202866333

Reply:
@baskaryan, thanks! I was talking to the client dev on our side and noticed your changes. We will likely update our docs instead of supporting 3.8